ui/apps/platform/src/Containers/VulnMgmt/widgets/TopRiskiestEntities.js

```diff
@@ -127,19 +236,137 @@ const TOP_RISKIEST_NODES = gql`
     }
 `;
 
+const TOP_RISKIEST_NODE_VULNS = gql`
+    query topRiskiestNodeVulns($query: String, $pagination: Pagination) {
+        results: nodes(query: $query, pagination: $pagination) {
+            id
+            name
+            vulnCounter: nodeVulnerabilityCounter {
+                all {
+                    total
+                    fixable
+                }
+                low {
+                    total
+                    fixable
+                }
+                moderate {
+                    total
+                    fixable
+                }
+                important {
+                    total
+                    fixable
+                }
+                critical {
+                    total
+                    fixable
+                }
+            }
+            priority
+            scan {
+                scanTime
+            }
+        }
+    }
+`;
+
 const getTextByEntityType = (entityType, data) => {
     switch (entityType) {
         case entityTypes.NODE:
-            return data.name;
+        case entityTypes.NODE_CVE:
+            return data.name.fullName || data.name;
```

Comment:
unforutnately had to do this bc of a render bug -- we should look into fixing unnecessary renders into the future